### PR TITLE
Update free-courses-pl.md

### PR DESCRIPTION
**Pull request by SuddenlyPineapple**
## What does this PR do?
Add resource(s)

### Description
Few simple Vue courses under JavaScript Section

### Why is this valuable (or not)?
These are simple and easy to understand

### How do we know it's really free?
You have access to whole course just by clicking in link

### For book lists, is it a book? For course lists, is it a course? etc.
It's a course 

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [x] Search for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction)

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

Please add `hacktoberfest-accepted` label upen merging ;)

### CODE REVIEW

File: courses/free-courses-pl.md [hunk 1]
```diff
@@ -82,6 +82,12 @@
 * [Programowanie w JavaScript od podstaw w 1 miesiąc](https://www.youtube.com/playlist?list=PLTs20Q-BTEMPRSzhrlAuu7yus1BuOLVrS)
 
 
+#### Vue
+
+* [FrontAndBack.pl - Kurs Vue w praktyce](https://frontandback.pl/tags/kurs-vue-w-praktyce/)
+* [WebMastah - Kurs Vue.js krok po kroku](https://webmastah.pl/kurs-vue-js-krok-po-kroku-wprowadzenie-i-hello-world/)
```
[the remark this answers is not in the document]
Comment (davorpa) [reply courses/free-courses-pl.md:88]: Remove patch
```suggestion
```

**Review by davorpa — CHANGES_REQUESTED** [review]
Apply patch https://github.com/EbookFoundation/free-programming-books/pull/6265#discussion_r739751805

File: courses/free-courses-pl.md [hunk 2]
```diff
@@ -82,6 +82,12 @@
 * [Programowanie w JavaScript od podstaw w 1 miesiąc](https://www.youtube.com/playlist?list=PLTs20Q-BTEMPRSzhrlAuu7yus1BuOLVrS)
 
 
+#### Vue
+
+* [FrontAndBack.pl - Kurs Vue w praktyce](https://frontandback.pl/tags/kurs-vue-w-praktyce/)
```
[the remark this answers is not in the document]
Comment (LuigiImVector) [reply courses/free-courses-pl.md:87]: it's a blog but the tags make it more organized, although to me there seems to be too little material

[the remark this answers is not in the document]
Comment (eshellman) [reply courses/free-courses-pl.md:87]: Ok, let's decline.

**Pull request merged**